services/src/main/java/org/keycloak/authorization/protection/permission/PermissionTicketService.java

```diff
@@ -226,6 +226,52 @@ public Response find(@QueryParam("scopeId") String scopeId,
                 .build();
     }
 
+    @Path("/count")
+    @GET
+    @Produces("application/json")
+    public Response getResourceCount(@QueryParam("scopeId") String scopeId,
+                                       @QueryParam("resourceId") String resourceId,
+                                       @QueryParam("owner") String owner,
+                                       @QueryParam("requester") String requester,
+                                       @QueryParam("granted") Boolean granted,
+                                       @QueryParam("returnNames") Boolean returnNames) {
+        StoreFactory storeFactory = authorization.getStoreFactory();
+        PermissionTicketStore permissionTicketStore = storeFactory.getPermissionTicketStore();
+        Map<String, String> filters = new HashMap<>();
+
+        if (resourceId != null) {
+            filters.put(PermissionTicket.RESOURCE, resourceId);
+        }
+
+        if (scopeId != null) {
+            ScopeStore scopeStore = storeFactory.getScopeStore();
+            Scope scope = scopeStore.findById(scopeId, resourceServer.getId());
+
+            if (scope == null) {
+                scope = scopeStore.findByName(scopeId, resourceServer.getId());
+            }
+
+            filters.put(PermissionTicket.SCOPE, scope != null ? scope.getId() : scopeId);
+        }
+
+        if (owner != null) {
+            filters.put(PermissionTicket.OWNER, getUserId(owner));
+        }
+
+        if (requester != null) {
+            filters.put(PermissionTicket.REQUESTER, getUserId(requester));
+        }
+
+        if (granted != null) {
+            filters.put(PermissionTicket.GRANTED, granted.toString());
+        }
+        long count = permissionTicketStore.count(filters, resourceServer.getId());
+
+        Map<String, Long> map = new HashMap<>();
```

Comment:
Just return the `Integer`. To keep aligned with other places. Although still not ideal, we are reviewing our REST guidelines.

Reply:
I was inspired by [the group count endpoint](https://github.com/keycloak/keycloak/blob/master/integration/admin-client/src/main/java/org/keycloak/admin/client/resource/UserResource.java). Will change it to `Integer`.